### PR TITLE
03_module: first module implementation

### PR DESCRIPTION
**Pull request by ASynenko**
Simple kernel module. Useless machine logic. No warnings from `checkpatch.pl`

### CODE REVIEW

File: 03_module/first_module.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	}
+
+	if (ansver == GREAT_ANSVER)
+		pr_warn("%s, you are correct! ansver is: %d",
+				user_name,
+				ansver);
+	else
+		pr_warn("%s, you are incorrect!", user_name);
+
+	return 0;
```
Comment (ReAlUA) [inline 03_module/first_module.c:37]: Implement task point 4.

**Review by ReAlUA — CHANGES_REQUESTED** [review]

**Comment by ReAlUA** [comment]
You can also add
```c
MODULE_PARM_DESC(user_name, "User name");
MODULE_PARM_DESC(answer, "User answer to the Ultimate Question");
```
for better `modinfo first_module.ko` output

File: 03_module/first_module.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+MODULE_DESCRIPTION("First module for Linux Kernel ProCamp");
+MODULE_LICENSE("Dual BSD/GPL");
+
+#define GREAT_ANSVER 42
+
+static char *user_name;
+module_param(user_name, charp, 0);
+
+static int ansver;
+module_param(ansver, int, 0);
```
Comment (ReAlUA) [inline 03_module/first_module.c]: *answer*

Comment (ASynenko) [reply 03_module/first_module.c]: fixed

File: 03_module/first_module.c (hunk 1 above)
Comment (ASynenko) [reply 03_module/first_module.c:37]: Done

File: 03_module/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# kbuild part of makefile
+obj-m := first_module.o
+
+else
+
+ifeq ($(TARGET),BBB) 
+
+REMDIR = /srv/nfs/busybox/home
+KDIR = ~/Projects/repos/linux/kernel/linux-stable/
```
Comment (ReAlUA) [inline 03_module/Makefile:15]: Removing of `?` may help when troubleshooting bot now
```bash
export TARGET=BBB
export KDIR=~/GL-LKT/repos/linux-stable/
make
```
does not work. Only this way is possible:
```bash
export TARGET=BBB
make KDIR=~/GL-LKT/repos/linux-stable/
```
It's not a big deal for this homework but read more about env / command line assignments priorities in makefiles.

**Review by ReAlUA — APPROVED** [review]
See comment.
You may merge this PR.

**Pull request merged**